src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -962,6 +962,22 @@
     <Field type="ThreadState" name="state" label="Thread State" />
   </Event>
 
+  <Event name="CPUTimeSample" category="Java Virtual Machine, Profiling" label="CPU Time Method Sample"
+    description="Snapshot of a threads state from the CPU time sampler. The throttle can be either an upper bound for the event emission rate, e.g. 100/s, or the cpu-time period, e.g. 10ms, with s, ms, us and ns supported as time units."
+    throttle="true" thread="false" experimental="true">
+    <Field type="StackTrace" name="stackTrace" label="Stack Trace" />
+    <Field type="Thread" name="eventThread" label="Thread" />
+    <Field type="boolean" name="failed" label="Failed" description="Failed to obtain the stack trace" />
+    <Field type="Tickspan" name="samplingPeriod" label="CPU Time Sampling Period"/>
+    <Field type="boolean" name="biased" label="Biased" description="The sample is biased towards the frame at safepoint" />
```

Comment:
"The sampled is biased towards the frame at safepoint". What does that mean? Are there three things involved? A sample, a frame and a safepoint? How are they related? See if you can clarify what you intend to communicate here.

Reply:
Maybe better: "The sample is safepoint-biased"